CampaignConfig/mdc2020_prolog.cfg

```diff
@@ -4,8 +4,8 @@ subgroup = highpro
 experiment = mu2e
 wrapper = file:///${FIFE_UTILS_DIR}/libexec/fife_wrap
 submitter = mu2e
-outdir_sim = /pnfs/mu2e/scratch/datasets/phy-sim/sim/mu2e/
-outdir_dts = /pnfs/mu2e/scratch/datasets/phy-sim/dts/mu2e/
+outdir_sim_scrath = /pnfs/mu2e/scratch/datasets/phy-sim/sim/mu2e/
```

Comment:
There is a typo here (scrath), I don't know if it matters

Reply:
Fixed